src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/LanguageServices/DateAndTimeEmbeddedLanguage.cs

```diff
@@ -18,12 +17,11 @@ internal class DateAndTimeEmbeddedLanguage : IEmbeddedLanguage
     {
         public readonly EmbeddedLanguageInfo Info;
 
-        public ISyntaxClassifier Classifier { get; }
+        public ISyntaxClassifier? Classifier { get; }
```

Comment:
```suggestion
        public ISyntaxClassifier? Classifier => null;
```

This makes it more clear that this value should not be set in a constructor

Reply:
Also worth a comment of why this doesn't have a syntax classifier